odk1-src/conf.py

```diff
@@ -66,7 +66,7 @@
 # The short X.Y version.
 version = '1.0'
 # The full version, including alpha/beta/rc tags.
-release = '1.0'
+#release = '1.0'
```

Comment:
I get a little nervous when something is commented out! Why was this removed?

Reply:
Sphinx has defined its own release command for latex for customization. I have renewed the command to take no arguments and display no release information. So, I commented this out since it was providing a release number argument for the release command.